testproject/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.conf.urls import url, include
+
+from testproject.views import login, logout
+
+
+urlpatterns = [
+
+    url(r'^i18n/', include('django.conf.urls.i18n')),
+    url(r'^login/', login, name='login'),
+    url(r'^logout/', logout, name='logout'),
```

Comment:
I've updated to reuse `django.contrib.auth.views.logout`. Not using their `login` view though, but I don't think that's a big deal in this context.